_component_design/loading-indicators.md

```diff
@@ -0,0 +1,163 @@
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-icon.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+A variety of spinner icon sizes are available to best suit the use case.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-size.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading spinner with text
+{: .hxSubSectionTitle}
+
+When text is contained inside a spinner icon, the spinner matches the size of the text. Therefore, spinner size is measured in *em*.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading text
+
+In certain instances, such as key-value pairs, or other list style patterns - a text label can be used without a spinner.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text-only.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Animation
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Animation can be used in loading indicators to reinforce that back-end systems are performing load operations. The following are best practice for spinner icon animations:
+
+- Use a simple animation.
+- Rotate icons 360 degrees in 0.8 seconds.
+- Always rotate icons clockwise.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-animation.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Layout
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Loading indicators are generally centered within their given component. The following are examples of how loading indicators inside various components of the UI:
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-layout.svg)
```

Comment:
Lets trim the whitespace on this SVG, currently it is hard to view. Consider also, splitting the image into multiple smaller images as another solution.

![image](https://user-images.githubusercontent.com/586682/32009939-044d5e46-b976-11e7-8521-86a35c08c339.png)